be/src/exec/workgroup/work_group.h

```diff
@@ -229,6 +229,7 @@ class WorkGroupManager {
     std::shared_mutex _mutex;
     std::unordered_map<int128_t, WorkGroupPtr> _workgroups;
     std::unordered_map<int64_t, int64_t> _workgroup_versions;
+    std::unordered_map<std::string, int128_t> _unique_workgroup_names;
```

Comment:
why need this unique name map？I think the unique id could identify one work group.